lib/broadway/caller_acknowledger.ex

```diff
@@ -21,4 +27,9 @@ defmodule Broadway.CallerAcknowledger do
   def ack({pid, ref}, successful, failed) do
     send(pid, {:ack, ref, successful, failed})
   end
+
+  @impl true
+  def configure({pid, ref}, _ack_data, _options) do
+    send(pid, {:configure, ref})
```

Comment:
We should probably pass the options upstream. It would be useful for testing that configure is being properly set.

Reply:
Done!